File: taskrouter/v1/worker_channel/fetch-default/fetch-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

worker_channel = @client.taskrouter
                        .workspaces('WSaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
                        .workers('WKaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
                        .worker_channels('WCaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
                        .fetch

puts worker_channel.sid

File: studio/v1/engagement_context/fetch-default/fetch-default.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$engagement_context = $twilio->studio->v1->flows("FWaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
                                         ->engagements("FNaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
                                         ->engagementContext()
                                         ->fetch();

print($engagement_context.accountSid);